package.json

```diff
@@ -45,7 +45,8 @@
     "prepack-prepack": "node --stack_size=10000 --max_old_space_size=8096 ./bin/prepack.js ./lib/prepack-cli.js --out ./lib/prepack-cli.prepacked.js --compatibility node-cli --mathRandomSeed rnd",
     "validate": "npm run build && npm run build-scripts && npm run lint && npm run depcheck && npm run flow && npm test",
     "prepublish": "npm run build",
-    "depcheck": "babel-node scripts/detect_bad_deps.js"
+    "depcheck": "babel-node scripts/detect_bad_deps.js",
+    "prettier": "prettier --write --single-quote --jsx-bracket-same-line --trailing-comma all --print-width 120 \"+(src|scripts)/**/*.js\""
```

Comment:
By far most changes are around turning strings to single-quote. This makes it really difficult to code review the actual changes. Is there a way to leave strings as they are? If not, I vote for starting out with double-quotes, and possibly changing that default later.